README.md

```diff
@@ -2,4 +2,162 @@
 
 [![Build Status](https://travis-ci.com/kaliber5/ecsy-babylon.svg?branch=master)](https://travis-ci.com/kaliber5/ecsy-babylon)
 
-WIP
+ecsy-babylon is an experimental implementation of [ECSY](https://ecsy.io/) in [babylon.js](https://www.babylonjs.com/).
+
+## Example
+
+In the spirit of learning-by-doing lets walk through how a simple babylon app would be converted to ecsy-babylon.
+
+### Vanilla babylon.js
+Consider the following code:
+```ts
+	//initialize the core elements
+	const canvas = document.getElementsByTagName("canvas")[0]
+	const engine = new BABYLON.Engine(canvas, true)
+	const scene = new BABYLON.Scene(engine)
+
+	// create objects to inhabit the scene
+	const camera = new BABYLON.ArcRotateCamera("camera", 
+		-Math.PI / 2, Math.PI / 2.5, 3, new BABYLON.Vector3(0, 0, 0), scene);
+	camera.attachControl(canvas, true);
+	const light = new BABYLON.HemisphericLight("light", 
+		new BABYLON.Vector3(0, 1, 0), scene);
+	const box = BABYLON.MeshBuilder.CreateBox("box", {}, scene);
+
+
+	// set variables
+	const freq = Math.PI //0.5 Hz
+	const amp = 0.1
+
+	// apply behaviour on each frame
+	engine.runRenderLoop(() => {
+		box.position.y = Math.sin(Date.now() * 0.001 * freq) * amp
+		scene.render()
+	})
+
+```
+
+We create a scene and then a box to bob up and down within it. At the moment our app is fairly simple but as it expands we will need to consider how to manage the increasing complexity.
+
+### ecsy-babylon
+
+ECS design will help us to write more organized apps by introducing some strict rules as to how they shall be structured:
+1. Conceptual elements of the app are organized into **entities**
+2. All game state and data are fields of **components**
+3. All behaviour exits in **systems**
+
+For more information please check out the [ECSY architecture docs](https://ecsy.io/docs/#/manual/Architecture).
+
+In keeping with rule 1, lets use ecsy-babylon to convert our existing objects to components on entities.
+
+```ts
+import * as EcsyBabylon from "ecsy-babylon";
```

Comment:
I would *slightly* prefer importing only the needed parts here, but that's not a deal-breaker for sure...